client/state/stats/lists/selectors.js

```diff
@@ -204,6 +204,14 @@ export function getSiteStatsViewSummary( state, siteId ) {
 		viewSummary[ years ][ months ].average = round( average, 0 );
 	} );
 
+	// With the current month, calculate the average based on the days passed so far in the month
+	const thisMonth = viewSummary[ moment().year() ][ moment().month() ];
```

Comment:
Do you think we can reduce the `moment()` calls to one in this case? And get the `month` and `year` from that single call, instead of calling `moment()` twice?